### DISCUSSION TITLE
Latest release sounds very promising... But is there any recommentations how to get it to work on a 6Gb RTX system?

### DISCUSSION THREAD

**Discussion by usigg3** [vladmandic/sdnext#2663] [Q&A]
Did a reinstall but it's incredibly slow. around 3-7 s/iteration if I'm lucky and then it takes forever to VAE decode... I really like this style of UI and had hoped to get back to this after trying both Comfy and Foocus but I can't seem to get it to run in a reasonable way.

**Comment by vladmandic** [comment]
Logs?
I don't know what's your platform or hardware or anything...

**Reply by usigg3** [reply]
I have a 2060 6 Gb. Tried it now with --lowvram in startup then I get around 2.7 sec/iter. for 624 by 1024... which is about half speed relative to Foocus for example. Is any of the options with compiling etc likely to improve things? It's with a SDXL model.

/Thanks

**Reply by vladmandic** [reply]
I still have no idea what your settings are, so can't recommend anything.

**Reply by usigg3** [reply]
32 GB RAM, Default settings, --lowvram in startup bat file, latest nVidia drivers so nothing special. Without --lowvram it's even slower around 7 sec/iteration.

**Reply by vladmandic** [reply]
Logs.
For example, depending on revision of 2060, fp16 may be flaky so sdnext auto disables it and forces fp32 which then gives you your 50% performance.
Without logs we can go like this forever.

**Reply by usigg3** [reply]
Is it the console log or is there a log file somewhere? 

/Thanks

**Reply by vladmandic** [reply]
yes, its in both. i'm not going to go into "how to grab logs" here.
